include/rocksdb/stats_history.h

```diff
@@ -11,19 +11,25 @@
 #include <map>
 #include <string>
 
-// #include "db/db_impl.h"
 #include "rocksdb/statistics.h"
 #include "rocksdb/status.h"
 
 namespace rocksdb {
 
 class DBImpl;
 
+// StatsHistoryIterator is the main interface for users to programmatically
+// access statistics snapshots that was automatically stored by RocksDB.
+// Depending on options, the stats can be in memory or on disk.
+// The stats snapshots are indexed by time that they were recorded, and each
+// stats snapshot is an std::map from individual stat name to an integer value
+// of the stat at the time of recording.
 class StatsHistoryIterator {
  public:
   StatsHistoryIterator() {}
   virtual ~StatsHistoryIterator() {}
 
+  // Return if the iterator is valid or now
```

Comment:
In my opinion, this falls into "do not be unnecessarily verbose or state the completely obvious".
If we want to add a comment here, it may make sense to describe what does invalid means here. I guess it's either it's already iterate to the end, or there is error happened?